portal-services/src/main/java/com/telecominfraproject/wlan/portal/controller/profile/ProfilePortalController.java

```diff
@@ -107,29 +108,21 @@ public ListOfProfiles getAllInSet(@RequestParam Set<Long> profileIdSet) {
 
     @RequestMapping(value = "/profile/forCustomer", method = RequestMethod.GET)
     public PaginationResponse<Profile> getForCustomer(@RequestParam int customerId,
+    		@RequestParam(required = false) ProfileType profileType,
             @RequestParam(required = false) List<ColumnAndSort> sortBy,
             @RequestParam(required = false) PaginationContext<Profile> paginationContext) {
 
-    	if(paginationContext == null) {
```

Comment:
Were these null checks not necessary anymore?

Reply:
all of the paths that do this just pass it through to the /api controller, which uses ProfileByCustomerRequestFactory. the Factory covers this, so the paginationContent can't be null before it's accessed